test/k8sT/Hubble.go

```diff
@@ -0,0 +1,183 @@
+// Copyright 2020 Authors of Cilium
```

Comment:
this file name is capitalized, is that intentional?

Reply:
Yes, to be consistent with the other files in that K8s folder. Not sure about the rationale though.

Reply:
We are not consistent anyway, let's use lowercase with underscores (no underscores for this file though), and move all our test spec files into this convention soon.

Reply:
Thanks for the feedback @nebril - it's now lowercase!